esphome/components/emporia_vue/sensor.py

```diff
@@ -73,30 +73,32 @@
         cv.GenerateID(): cv.declare_id(CTSensor),
         cv.Required(CONF_PHASE_ID): cv.use_id(PhaseConfig),
         cv.Required(CONF_INPUT): cv.enum(CT_INPUT),
-        cv.Optional(CONF_CALIBRATION, default=0.022): cv.zero_to_one_float,
     }
 )
 
 CONFIG_SCHEMA = (
     cv.Schema(
         {
             cv.GenerateID(): cv.declare_id(EmporiaVueComponent),
+            cv.Optional(
+                CONF_SENSOR_POLL_INTERVAL, default="240ms"
+            ): cv.positive_time_period_milliseconds,
             cv.Required(CONF_PHASES): cv.ensure_list(
                 {
                     cv.Required(CONF_ID): cv.declare_id(PhaseConfig),
                     cv.Required(CONF_INPUT): cv.enum(PHASE_INPUT),
+                    cv.Optional(CONF_CALIBRATION, default=0.022): cv.zero_to_one_float,
                     cv.Optional(CONF_VOLTAGE): sensor.sensor_schema(
-                        UNIT_VOLT,
-                        ICON_EMPTY,
-                        1,
-                        DEVICE_CLASS_VOLTAGE,
-                        STATE_CLASS_MEASUREMENT,
+                        unit_of_measurement=UNIT_VOLT,
```

Comment:
This avoids setting the icon parameter, so that it inherits the default (which looks like a sine wave in Home Assistant)

Reply:
Good idea 👍